app/src/main/java/mozilla/lockbox/view/SettingViewHolder.kt

```diff
@@ -24,40 +25,46 @@ abstract class SettingViewHolder(override val containerView: View) :
 }
 
 class TextSettingViewHolder(val view: View) : SettingViewHolder(view) {
-    var title: String? = null
-        set(value) {
+    var title: Int? = null
+        set(@StringRes value) {
             field = value
-            view.description.text = value
+            value?.let {
+                view.description.text = view.resources.getString(it)
+            }
```

Comment:
All these `TextView`s can use `view.title.setText(value)` here.

Also consider: for bonus extra, add an `emptyString` in `R.string`. This might not be the intended outcome however, re: placeholder texts.